Gopkg.toml

```diff
@@ -63,7 +63,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/percona-backup-mongodb"
-  version = "0.3.0"
+  branch = "master"
```

Comment:
Are you sure we want to vendor a master branch?

Reply:
for now its the only branch that works